tests/Microsoft.DotNet.XHarness.Android.Tests/AdbRunnerTests.cs

```diff
@@ -0,0 +1,265 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using Microsoft.DotNet.XHarness.Android.Execution;
+using Microsoft.Extensions.Logging;
+
+using Moq;
+using Xunit;
+
+namespace Microsoft.DotNet.XHarness.Android.Tests
+{
+    public class AdbRunnerTests : IDisposable
+    {
+        private static readonly string s_scratchAndOutputPath = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
+        private static readonly string s_adbPath = Path.Combine(s_scratchAndOutputPath, "adb");
+        private static string s_currentDeviceSerial = "";
+        private readonly Mock<ILogger> _mainLog;
+        private readonly Mock<IAdbProcessManager> _processManager;
+        private Dictionary<Tuple<string, string>, int> _fakeDeviceList;
```

Comment:
nit: This one can be made `readonly` and the new assigned here since class is instantiated for every test again